base/socket.jl

```diff
@@ -21,9 +21,9 @@ end
 
 function IPv4(host::Integer)
     if host < 0
-        error("IP address must not be negative")
+        throw(ArgumentError("IPv4 address must not be ≥ 0"))
```

Comment:
@GlenHertz thanks for reading through.  "must be positive" I think is subjectively better than "not negative", I'll push an update.

@IainNZ I have no idea.